soc/nxp/imxrt/imxrt6xx/Kconfig.defconfig

```diff
@@ -65,3 +65,16 @@ config SYS_CLOCK_HW_CYCLES_PER_SEC
 endif # CORTEX_M_SYSTICK
 
 endif # SOC_MIMXRT685S_CM33
+
+if SOC_MIMXRT685S_HIFI4
+config SOC_TOOLCHAIN_NAME
+	string
+	default "nxp_rt600_adsp"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 198000000
```

Comment:
needs to come from dts property

Reply:
for right now we are not doing this on NXP platforms, I agree strongly but it is not to be solved on this PR

Reply:
Can you plan and submit a fix for adding support for this to all NXP SoC's in the near future? Will allow it for now on this PR, will begin enforcing NACKs on this in a couple of weeks

Reply:
@nordicjm I'm going to create an enhancement request ticket to track this. Can you provide a link to where the require is coming from so I can include it in the ticket?